src/Storefront/Resources/app/storefront/src/scss/skin/shopware/component/_breadcrumb.scss

```diff
@@ -7,14 +7,30 @@ https://getbootstrap.com/docs/5.2/components/breadcrumb
 */
 
 .breadcrumb {
-    .breadcrumb-placeholder {
-        margin: 0 $spacer-sm;
+    @if not feature('v6.8.0.0') {
+        .breadcrumb-placeholder {
+            margin: 0 $spacer-sm;
+        }
+
+        svg {
+            height: $font-size-base;
+            width: auto;
+            top: 3px;
+        }
     }
 
-    svg {
-        height: $font-size-base;
-        width: auto;
-        top: 3px;
+    .breadcrumb-item + .breadcrumb-item:before {
```

Comment:
Is this already the new divider? Could we use `--bs-breadcrumb-divider` for this as in their docs? https://getbootstrap.com/docs/5.3/components/breadcrumb/#example

Reply:
@tobiasberge yes, that we could also do, I tried to replicate the current look, as the default Bootstrap separator is `/` which looks different. 

However I am not really a design person, so it could also be replaced by the `/` :-)

Reply:
Could we put our current arrow svg as encoded in `--bs-breadcrumb-divider`? I'm not the biggest fan of those encoded SVGs either, but if thats their API to modify this. Then we would use the Bootstrap structure and keep the current look.

Reply:
I think thats fine to do this, in the Bootstrap docs it is also shown like this with an SVG: 
<img width="783" height="153" alt="image" src="https://github.com/user-attachments/assets/7ad6b0c0-6ad2-4adf-a2d7-c7ca6d763a3a" />

We can put the SVG string itself into a block or variable, so it can still be overwritten if that is wanted.

Reply:
So, now I implemented it as bootstrap default.

However to align the icons properly I needed to adjust the CSS a bit. Not sure if we want to do that, or if there is another solution?

<img width="1644" height="364" alt="image" src="https://github.com/user-attachments/assets/202cb223-b910-482d-9c71-5aec987baa27" />